docs/site/migration/auth/example.md

```diff
@@ -636,6 +559,56 @@ $ lb4 repository
 ? Please select the repository base class DefaultCrudRepository (Legacy juggler bridge)
 ```
 
+Create relations:
+
+- `Project` belongsTo `User`
```

Comment:
in the instruction, it changes the project.model.ts to add the `belongsTo` relation with `User`.  Is it a duplicate in either place?

Reply:
deleted the manual creation ones :p